core/jreleaser-engine/src/main/java/org/jreleaser/engine/context/ModelConfigurer.java

```diff
@@ -87,6 +87,24 @@ public static void configure(JReleaserContext context) {
         }
     }
 
+    private static void autoConfigureBitbucketCloud(JReleaserContext context, Repository repository) {
+        BaseReleaser<?, ?> service = context.getModel().getRelease().getReleaser();
+
+        if (null != service) {
+            if (!(service instanceof BitbucketCloudReleaser)) {
+                // TODO: just following GitHub trail, will figure out what this is for later, but it is intriguing
```

Comment:
err : use a prefix for your TODOs Haz ... `// TODO: viqueen - ` or `// TODO: TRACKER-123 - ` 
I forgot how to dev in the span of one week wow